app/models/subheading.rb

```diff
@@ -38,6 +38,10 @@ def to_s
     formatted_description || description
   end
 
+  def declarable?
```

Comment:
:tada:

Reply:
I believe this comes back in the api but why not

Reply:
Actually, should we just let the api tell us this? Every goods nomenclature does

Reply:
https://github.com/trade-tariff/trade-tariff-backend/blob/main/app/serializers/api/v2/subheadings/subheading_serializer.rb#L18-L20

Reply:
Ah, ok - will drop this